kmk/extensions/led.py

```diff
@@ -26,12 +35,16 @@ def __init__(
         user_animation=None,
         val=100,
     ):
+        self._leds = []
         try:
-            self._led = pulseio.PWMOut(led_pin)
+            if not isinstance(led_pin, list):
+                led_pin = [led_pin]
+            for pin in led_pin:
+                self._leds.append(pwmio.PWMOut(pin))
```

Comment:
to be compatible with generators, this could use a try/except of trying to iterate over `led_pin`, and on failure, assign the iterator to `[led_pin]`, a-la https://stackoverflow.com/a/1952655

```suggestion
            try:
                pins_iter = iter(led_pin)
            except TypeError:
                pins_iter = [led_pin]
                
            for pin in pins_iter:
                self._leds.append(pwmio.PWMOut(pin))
```